PKGBUILDS/phosh/phosh/PKGBUILD

```
@@ -9,7 +9,7 @@ license=('GPL')
 depends=('gtk3' 'libhandy>=1.1.90' 'gnome-desktop' 'gnome-session' 'gnome-shell' 'upower' 'libpulse'
          'gcr' 'feedbackd' 'libnm' 'phoc' 'callaudiod')
 makedepends=('meson' 'ninja' 'git')
-_commit="e9651c72a6c3033597d40b135c230262a2bf4cb6"  # tags/v0.17.0
+_commit="5fe9aea775b9198eed25fb3b3bcd9827ae40a785" # tags/v0.20.0
```

```suggestion
_commit="5fe9aea775b9198eed25fb3b3bcd9827ae40a785"  # tags/v0.20.0
```

Nitpick: I think we should get the blank space back.